lib/src/test/resources/generators/play-2-standalone-json-spec-quality.txt

```diff
@@ -489,23 +489,32 @@ package com.gilt.quality.v0.models {
     import play.api.libs.json.JsString
     import play.api.libs.json.Writes
     import play.api.libs.functional.syntax._
+    import org.joda.time.format.DateTimeFormat
+    import org.joda.time.format.DateTimeFormatterBuilder
     import com.gilt.quality.v0.models.json._
 
+    private[v0] val DateTimeFormatter =
+      new DateTimeFormatterBuilder().append(
+        org.joda.time.format.ISODateTimeFormat.dateTime.getPrinter,
+        Array(
+          DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZZ").getParser,
+          DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").getParser
+        )
+      ).toFormatter
```

Comment:
Joda time retains the original timezone if we do this. We may or may not need this, but I think it's still nice to have.